frontend/src/components/Pages/RobotPage/RobotPage.tsx

```diff
@@ -70,26 +70,41 @@ export function RobotPage() {
             clearInterval(intervalId)
         }
     }, [selectedRobot])
+
     return (
         <>
             <Header page={'robot'} />
             <StyledRobotPage>
                 <BackButton />
-                <Typography variant="h1">{selectedRobot?.name + ' (' + selectedRobot?.model.type + ')'}</Typography>
-                <RobotInfo>
-                    <RobotImage height="350px" robotType={selectedRobot?.model.type} />
-                    <VerticalContent $alignItems="start">
-                        <BatteryStatusView
-                            itemSize={48}
-                            battery={selectedRobot?.batteryLevel}
-                            batteryStatus={BatteryStatus.Normal}
-                        />
-                        <PressureStatusView itemSize={48} pressure={selectedRobot?.pressureLevel} />
-                        <RobotStatusChip status={selectedRobot?.status} />
-                    </VerticalContent>
-                </RobotInfo>
                 {selectedRobot && (
                     <>
+                        <Typography variant="h1">
+                            {selectedRobot?.name + ' (' + selectedRobot?.model.type + ')'}
+                        </Typography>
+                        <RobotInfo>
+                            <RobotImage height="350px" robotType={selectedRobot?.model.type} />
+                            <VerticalContent $alignItems="start">
+                                <BatteryStatusView
+                                    itemSize={48}
+                                    battery={selectedRobot?.batteryLevel}
+                                    batteryStatus={BatteryStatus.Normal}
+                                />
+                                {
+                                    <PressureStatusView
```

Comment:
```suggestion
                                   {selectedRobot.model.upperPressureWarningThreshold && <PressureStatusView
```

Reply:
Is this resolved?

Reply:
I am dismissing the review from you as it has been resolved and pushing it in :)